### PR TITLE
Feat/show all apys

### PR DESCRIPTION
**Pull request by octocat512**

### CODE REVIEW

**Comment by octocat512** [comment]
- [ ] search pool feature
- [ ] review UI/UX

File: src/app/components/trade/LiquidityPoolsPanel.tsx [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <BoxPanel bg="bg2" mb={10}>
+      <Flex alignItems="center" justifyContent="space-between" mb={5}>
+        <Flex alignItems="center">
+          <Typography variant="h2" mr={4}>
+            <Trans>Liquidity pools</Trans>
+          </Typography>
+
+          <ChartControlGroup>
+            <ChartControlButton type="button" onClick={() => handleSwitch(0)} active={value === 0}>
+              <Trans>Your Pools</Trans>
```
Comment (parrot9design) [inline src/app/components/trade/LiquidityPoolsPanel.tsx:39]:
```suggestion
              <Trans>Your pools</Trans>
```

File: src/app/components/trade/LiquidityPoolsPanel.tsx [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+            <ChartControlButton type="button" onClick={() => handleSwitch(1)} active={value === 1}>
+              <Trans>All pools</Trans>
+            </ChartControlButton>
+          </ChartControlGroup>
+        </Flex>
+
+        <SearchInput
+          type="text"
+          id="pool-search-input"
+          placeholder={t`Search pools ...`}
```
Comment (parrot9design) [inline src/app/components/trade/LiquidityPoolsPanel.tsx:50]:
```suggestion
          placeholder={t`Search pools...`}
```

**Comment by parrot9design** [comment]
- [ ]  If you're signed out or haven't supplied liquidity, you should still be able to see the liquidity pools section when the `Supply liquidity` tab is active. In those states, it should look like this:
169448292-65ac4c5c-9637-4649-9268-b164d33f55ec

- [ ] When in the `All pools` view, no details are loading currently

- [ ] We should use the same BALN/Fee APYs (APRs?) as in https://github.com/balancednetwork/balanced-network-info/pull/83

- [ ] It might be better to merge this in with https://github.com/balancednetwork/balanced-network-interface/pull/770 so they can be released at the same time. The `Your pools` view will change a little to accommodate for LP token staking.

**Comment by octocat512** [comment]
@parrot9design please review the UX again.

**Comment by parrot9design** [comment]
- [ ] Let's remove the participants section to give the data more room to breathe. It feels bit too crowded at the moment, and won't help people make liquidity decisions.
- [ ] Let's add the BALN + fee APYs (and related tooltip) that are now on the Stats page.
- [ ] Let's add a hover effect to the `All pools` view (like shown [here](https://balanced.network/prototype-dev/trade/)), and make it so that if you click a pool, it loads in the supply liquidity section. We can add the same behaviour to the `Your pools` view once it includes the LP token staking UI.

In the design, we didn't include the total row at the bottom as we expected to add pagination (with 10 on each page) to keep it tidy and reduce the burden of API calls. Open to keeping it as you've implemented it, though. Which would you prefer?

**Comment by BennyOptions** [comment]
Should be finished and merged within a few days.

**Comment by octocat512** [comment]
> * [ ]  Let's remove the participants section to give the data more room to breathe. It feels bit too crowded at the moment, and won't help people make liquidity decisions.
> * [ ]  Let's add the BALN + fee APYs (and related tooltip) that are now on the Stats page.
> * [ ]  Let's add a hover effect to the `All pools` view (like shown [here](https://balanced.network/prototype-dev/trade/)), and make it so that if you click a pool, it loads in the supply liquidity section. We can add the same behaviour to the `Your pools` view once it includes the LP token staking UI.
> 
> In the design, we didn't include the total row at the bottom as we expected to add pagination (with 10 on each page) to keep it tidy and reduce the burden of API calls. Open to keeping it as you've implemented it, though. Which would you prefer?

I prefer to remove the total row. I'd like to show all pools on one page for now as there aren't many pools. @parrot9design  I hope to implement the pagination in the future as the number of pools grows.

**Comment by parrot9design** [comment]
>I prefer to remove the total row. I'd like to show all pools on one page for now as there aren't many pools. @parrot9design I hope to implement the pagination in the future as the number of pools grows.

Works for me, thanks @octocat512!

**Comment by parrot9design** [comment]
- [ ] When signed in on mobile without any liquidity, it currently looks like this:

IMG_F5ECC56A6C63-1
It should default to the `All pools` view, and not give the option to switch between views.

- [ ] We should also fix up the heading/view selector to avoid wrapping over 2 lines:

IMG_F5ECC56A6C63-1
It might be easiest to float the view selectors underneath the heading at smaller screen sizes.
 

- [ ] When `All pools` is active on mobile, it's not clear that you can scroll horizontally to see more details:
IMG_B2C59946E695-1
At smaller screen sizes, we could bring the APY column closer so that you can see more info, like with the exchange section on the Stats page:
IMG_09F4DB251CE1-1

- [ ] Let's make it possible to click a pool in the `All pools` view to make that pair active in the supply liquidity section for easier action.

**Comment by BennyOptions** [comment]
wait for refactor

**Comment by BennyOptions** [comment]
@octocat512 code refactor has been pushed so this should not be blocked anymore I think

**Comment by BennyOptions** [comment]
@octocat512 can you give an update on this PR? When do you expect to complete it?

**Comment by BennyOptions** [comment]
Note - will aim to finish the PR this week

**Comment by BennyOptions** [comment]
@parrot9design let's do one more review before merge

**Comment by parrot9design** [comment]
- When you're in a state to only see the `All pools` tab, hide the `All pools` button:
Screen Shot 2022-08-10 at 2 01 39 PM

The slider isn't appearing for the ICX pool currently:
Screen Shot 2022-08-10 at 2 05 41 PM

The data isn't loading in for the liquidity pools section and I'm getting the "Balanced DEX: Rewards distribution in progress" message when I try to complete any transaction, so I can't test further. @octocat512 would Sejong be more reliable for testing this?

**Comment by parrot9design** [comment]
Let's move the `Your pool` and `All pool` labels to beside the heading, like it is in the second image:
Screen Shot 2022-08-17 at 8 56 04 AM
Screen Shot 2022-08-17 at 8 58 32 AM

I would like to add the search bar and pagination, as the list is getting a bit long (17 pools currently), but we can save that for a future PR if you'd prefer to close this one out sooner.

---

At mobile, let's bring the APY column a bit closer so you can see that there's more info to view if you scroll horizontally. At the moment it just looks like a list:
IMG_D564F3487A6D-1
(taken on an iPhone 13 Pro)

---

The top section doesn't match the details in the liquidity pool section for any pool, both before and after LP tokens have been staked: 
Screen Shot 2022-08-17 at 9 10 34 AM
Screen Shot 2022-08-17 at 9 22 37 AM

It seems to reset the numbers when you click the `Supply` button, then doesn't update them after the transaction has been completed.


---

When testing the above issue, I found myself wishing the top section would also change when expanding a pool in the  `Your pools` section. Can we add that ability?

**Comment by parrot9design** [comment]
@octocat512 can you please set this to mainnet for easier testing? Berlin is too unreliable, as it doesn't load in the liquidity pool info.

**Comment by hetfly** [comment]
@parrot9design set to mainnet

**Comment by parrot9design** [comment]
If you click the BALN/bnUSD pool while in the `All pools` view, the page goes white.

Is it possible to prevent the content from jumping around when you click on a pool (caused by updating the `Supply liquidity` pair)? It's not so bad at regular screen sizes, but on mobile you have to look around too much to find the row you just tapped. Can be solved separately if it's beyond the scope of this PR, as long as we don't delay it too long.

Otherwise, this looks good to go from a UX perspective.

**Comment by BennyOptions** [comment]
@parrot9design @hetfly I just used the app on this branch and found no issues. Clicked around, supplied liquidity, traded, interacted with loan, etc. All worked well.

**Review by parrot9design — APPROVED** [review]

**Comment by BennyOptions** [comment]
@hetfly it appears there are conflicts that need to be resolved here as well.

**Comment by hetfly** [comment]
@BennyOptions resolved

**Pull request merged**